nucypher/characters.py

```diff
@@ -658,10 +734,26 @@ def grant(self, bob, uri, m=None, n=None, expiration=None, deposit=None, ursulas
         # Users may decide to inject some market strategies here.
         #
         # TODO: 289
+
+        # If we're federated only, we need to block to make sure we have enough nodes.
+        if self.federated_only and len(self._known_nodes) < n:
+            good_to_go = self.block_until_number_of_known_nodes_is(n, learn_on_this_thread=True)
+            if not good_to_go:
+                raise ValueError(
+                    "To make a Policy in federated mode, you need to know about\
```

Comment:
I suspect, this will produce lots of spaces which you use for alignment, in the string every time you make a new line (or, at least, it does for me when I repeat this pattern in terminal).
Maybe better:
```
    raise ValueError(
        "Starting the message,"
        "continuing the message")
```